packages/next/build/webpack-config.ts

```diff
@@ -115,6 +116,11 @@ export default async function getBaseWebpackConfig(
               dev ? `next-dev.js` : 'next.js'
             )
           ),
+        [path.join(
+          CLIENT_STATIC_FILES_PATH,
+          buildId,
```

Comment:
The polyfill chunk shouldn't change between builds. This should likely exist in the `chunks/` directory and use its content hash.

See the approach I took in https://github.com/zeit/next.js/pull/9244, maybe we can do a hybrid.

Reply:
Thats a very good point. I think a hybrid approach is the way to go